manim/mobject/mobject.py

```diff
@@ -925,22 +917,47 @@ def construct(self):
         :class:`~.UpdateFromFunc`
         """
 
+        # Test if updater is time based and whether to use time difference
+        parameters = list(get_parameters(updater).keys())
+        time_based = len(parameters) > 1
+        if time_based:
+            if parameters[1] not in ["t", "time"] and pass_relative_times is None:
+                pass_relative_times = True
+            pass_relative_times = bool(pass_relative_times)
```

Comment:
again, I don't particularly like this hard coded check for the second argument, but with a default of `True`, you can simplify the logic to only check for if it should be set to `False`
```suggestion
            if parameters[1] in ["t", "time"]:
                pass_relative_times = False
```